lib/utils/docker.coffee

```diff
@@ -130,7 +130,7 @@ parseBuildArgs = (args) ->
 		args = [ args ]
 	buildArgs = {}
 	args.forEach (arg) ->
-		pair = /^([^\s]+?)=(.*)$/.exec(arg)
```

Comment:
It would be good to hear from @dfunckt too, as I think I saw his name as the last person to touch this line of the code. :-)